lib/network/kmip/Client.ts

```diff
@@ -364,7 +363,7 @@ export default class Client {
             KMIP.Structure('Template-Attribute', attributes),
         ], (err, response) => {
             if (err) {
-                const error = arsenalErrorKMIP(err);
+                const error = arsenalErrorKMIP(JSON.stringify(err));
```

Comment:
do we really want to stringify: should not just be `err.message` ?
and/or is there a risk of leaking details to the (s3 api) caller?